bin/install

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-brew install --devel go
+brew install go
```

Comment:
you may be able to do something like `go@1.8` to lock it to a specific version instead of just the latest (I know `--devel` was even more dangerous...)